<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WxSphereCropperControls</class>
 <widget class="QWidget" name="WxSphereCropperControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <widget class="QPushButton" name="buttonEnd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>EndInteraction</string>
   </property>
  </widget>
  <widget class="QmitkDataStorageComboBox" name="boundingShapeSelector">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCreateNewBoundingBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Do Cropping</string>
   </property>
   <property name="text">
    <string>StartInteraction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Do Cropping</string>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxInsideOut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>InsideOut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUndo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Do Cropping</string>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRedo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Do Cropping</string>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
